src/lexer.l

```diff
@@ -542,8 +600,8 @@ UNION         ?i:union
 <VLOG>"inout"            { return tINOUT; }
 <VLOG>"reg"              { return tREG; }
 <VLOG>"always"           { return tALWAYS; }
-<VLOG>"posedge"          { return tPOSEDGE; }
-<VLOG>"negedge"          { return tNEGEDGE; }
+<VLOG>{POSEDGE}          { return tPOSEDGE; }
+<VLOG>{NEGEDGE}          { return tNEGEDGE; }
```

Comment:
This isn't correct because it makes `posedge` and `negedge` case-insensitive in Verilog. You can leave it as it is and use `{POSEDGE}` and `{NEGEDGE}` for SDF only. I think you can also write `<SDF>?i:POSEDGE` directly in the place where it's used (not tested).